shared/util/saga.js

```diff
@@ -83,13 +83,28 @@ function safeTakeLatest (pattern: string | Array<any> | Function, worker: Functi
   return takeLatest(pattern, wrappedWorker, ...args)
 }
 
+// take on pattern. If pattern happens while the original one is running just ignore it
+function* safeTakeLatestIgnoreMultiple (pattern: string | Array<any> | Function, worker: Function, ...args: Array<any>): any {
+  const task = yield fork(function* () {
+    let lastTask
+    while (true) {
+      const action = yield take(pattern)
+      if (!lastTask || !lastTask.isRunning()) {
+        lastTask = yield fork(worker, ...args.concat(action))
```

Comment:
Also this should do the error-catching `wrappedWorker` stuff like the others (pull out into a helper?) to fit the safe* naming.